vmdb/spec/support/automation_spec_helper.rb

```diff
@@ -56,4 +56,19 @@ def default_ae_model_attributes(attrs = {})
       :enabled       => true,
       :instance_name => 'instance1')
   end
+
+  def send_ae_request_via_queue(args)
+    MiqQueue.put(:role        => 'automate',
+                 :class_name  => 'MiqAeEngine',
+                 :method_name => 'deliver',
+                 :args        => [args])
+  end
+
+  def deliver_ae_request_from_queue
+    q = MiqQueue.all.detect { |item| item.state == 'ready' && item.class_name == "MiqAeEngine" }
```

Comment:
I'd really like to get away from using sql with the `MiqQueue` - could you introduce `EmsSpecHelper.remote_guid_miq_server_zone`?